<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/resources.qrc">
    <normaloff>:/icons/qneurolots.ico</normaloff>:/icons/qneurolots.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenBlueConfig"/>
    <addaction name="actionOpenXMLScene"/>
    <addaction name="actionOpenSWCFile"/>
    <addaction name="actionCloseData"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionUpdateOnIdle"/>
    <addaction name="actionPaintNeurons"/>
    <addaction name="actionShowFPSOnIdleUpdate"/>
    <addaction name="actionWireframe"/>
    <addaction name="actionBackgroundColor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenBlueConfig"/>
   <addaction name="actionOpenXMLScene"/>
   <addaction name="actionCloseData"/>
   <addaction name="actionOpenSWCFile"/>
   <addaction name="separator"/>
   <addaction name="actionWireframe"/>
   <addaction name="actionCloseData"/>
   <addaction name="actionBackgroundColor"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionUpdateOnIdle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update on idle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="icon">
    <iconset resource="../qrc/resources.qrc">
     <normaloff>:/icons/colorpicker.png</normaloff>:/icons/colorpicker.png</iconset>
   </property>
   <property name="text">
    <string>Background color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionOpenBlueConfig">
   <property name="icon">
    <iconset resource="../qrc/resources.qrc">
     <normaloff>:/icons/open_bc.png</normaloff>:/icons/open_bc.png</iconset>
   </property>
   <property name="text">
    <string>Open BlueConfig</string>
   </property>
   <property name="toolTip">
    <string>Open BlueConfig</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionOpenXMLScene">
   <property name="icon">
    <iconset resource="../qrc/resources.qrc">
     <normaloff>:/icons/open_xml.png</normaloff>:/icons/open_xml.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open XML Scene</string>
   </property>
   <property name="toolTip">
    <string>Open XML Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionCloseData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/resources.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close Data</string>
   </property>
   <property name="toolTip">
    <string>Close Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpenSWCFile">
   <property name="icon">
    <iconset resource="../qrc/resources.qrc">
     <normaloff>:/icons/open_swc.png</normaloff>:/icons/open_swc.png</iconset>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open SWC</string>
   </property>
   <property name="toolTip">
    <string>Open SWC</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionShowFPSOnIdleUpdate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show FPS on idle update</string>
   </property>
   <property name="toolTip">
    <string>Show FPS on idle update</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/resources.qrc">
     <normaloff>:/icons/wireframe.png</normaloff>:/icons/wireframe.png</iconset>
   </property>
   <property name="text">
    <string>Show wireframe</string>
   </property>
   <property name="toolTip">
    <string>Show wireframe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>  
  <action name="actionPaintNeurons">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
    <string>Paint neurons</string>
    </property>
    <property name="toolTip">
    <string>Paint neurons</string>
    </property>
   </action>  
 </widget>
 <resources>
  <include location="../qrc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
